scripts/codegen

```diff
@@ -29,4 +29,4 @@ if [ ! -f "$GENGO_BOILERPLATE" ]; then
     git clone https://github.com/kubernetes/gengo "${GENGO_BOILERPLATE_DIR}"
 fi
 
-"${CODEGEN_SCRIPT}" all github.com/submariner-io/submariner/pkg/client github.com/submariner-io/submariner/pkg/apis submariner.io:v1
+"${CODEGEN_SCRIPT}" client,deepcopy,informer,lister github.com/submariner-io/submariner/pkg/client github.com/submariner-io/submariner/pkg/apis submariner.io:v1
```

Comment:
Why not `all`?

Reply:
Because `all` is deprecated (I’ll update the commit message).